### DISCUSSION TITLE
3rd way to detect changes?

### DISCUSSION THREAD

**Discussion by Coffee2JS** [xaviergonz/mobx-keystone#222] [Q&A] [answered]
In the docs it says:

> As seen in the previous snapshots section, any change made to a tree node will generate a new snapshot, but this is only one of the three possible ways mobx-keystone offers to detect changes. The second way is "patches".

What's the 3rd way?

And, kudos for good work, this is a great library 👍

**Comment by xaviergonz** [comment, marked as answer]
I guess the third one would be mobx reactions :)
